packages/graphql_server/package.json

```diff
@@ -11,9 +11,11 @@
   "scripts": {
     "dev": "concurrently -c 'magenta' 'tsc -w' '$npm_package_config_env_dev && nodemon dist/index.js'",
     "build": "tsc",
-    "serve": "$npm_package_config_env_dev && node dist/index.js"
+    "serve": "$npm_package_config_env_dev && node dist/index.js",
+    "update-types": "npx supabase gen types typescript --project-id \"yipnhkmklmbuxjwzpipg\" > types/supabase.d.ts"
   },
   "dependencies": {
+    "@supabase/supabase-js": "^2.23.0",
```

Comment:
if this dependency is added here, then remove the supabase dependency, delete package-lock.json/node_modules and run npm i again

Reply:
we will use supabase-js dependency to fix possible former deployment bug